reportportal/templates/analyzer-statefulset.yaml

```diff
@@ -58,6 +58,8 @@ spec:
           value: "analyzer-default"
         - name: ES_HOSTS
           value: "{{ .Values.elasticsearch.endpoint }}"
+        - name: UWSGI_WORKERS
```

Comment:
Please add a condition for those who do not have this value in `values.yaml`, as here: https://github.com/reportportal/kubernetes/blob/master/reportportal/templates/analyzer-statefulset.yaml#L47

Reply:
I added check of Values.serviceanalyzer.uwsgi_workers